### ISSUE TITLE
items.dat encoder

### ISSUE THREAD

**Issue by DEERUX** [GrowtopiaNoobs/GrowtopiaServer#1150]
https://github.com/iProgramMC/Growtopia_ItemsDatEncoder/ enjoy skids, also decoder is in his git too.
made by gtv3 team 4 months ago. now published.
have fun skids!

**Comment by ama6nen** [comment]
Ness joined in November, sorry the months don't add up

**Comment by DEERUX** [comment]
we had it w/o the names 5 months ago tho.

**Comment by ama6nen** [comment]
Untrue

**Comment by iProgramMC** [comment]
@DEERUX lying again, WHY?! I made the encoder in October, THREE months ago not FIVE!!

**Comment by DEERUX** [comment]
@iProgramMC without names? also it doesn't mean that I didnt have one back then since I had made an items.dat editor 5.5 months ago it could edit everything and just rewrite bytes unchanged except for certain stuff such as itemKind. I actually a decent amount of stuff already before you joined but yeah you did the actual / fully working encoder and a lot of packet reversing *after* you learned how they work.

**Comment by iProgramMC** [comment]
uh no? you hired me **BECAUSE** I made the items.dat decoder for you

**Comment by DEERUX** [comment]
I had an items.dat editor which just was able to rewrite a couple bytes as I said (basically like a hex editor version except slightly advanced).
Also yeah I guess 3months+ isnt bad either... prob i was just wrong when it was about the time, but in **ONE** thing im pretty sure: we had it before most people had it.

**Comment by iProgramMC** [comment]
ok

**Comment by ama6nen** [comment]
Lies exposed part billion

**Comment by iProgramMC** [comment]
lol ok

**Comment by DEERUX** [comment]
@iProgramMC no I meant I had this hex kinda thing like hadi / cmd inb4

**Comment by iProgramMC** [comment]
Care to show on Discord?

**Comment by DEERUX** [comment]
it was literally with xvi32 dont u remember?

**Comment by iProgramMC** [comment]
xvi32 is a text editor that I showed you don't lie

**Comment by DEERUX** [comment]
we used it to modify the items.dat's bytes?

**Comment by iProgramMC** [comment]
You did, but I think I showed it to you.

**Comment by DEERUX** [comment]
yes exactly i was disappointed of hxd and you showed me xvi32 and we did it together back then already, thats why i also said "made by *gtv3 team*"

**Comment by iProgramMC** [comment]
Ok but that wasn't 5 mo ago

**Comment by DEERUX** [comment]
for real? i even got the feeling that it was 6months ago lol

**Comment by iProgramMC** [comment]
lol ok
